tests/e2e/transfer/transfer.test.ts

```diff
@@ -216,7 +234,7 @@ test.skip("Transfer page using imported account", async () => {
     await page.click("button >> text=Cancel");
   });
 
-  test("Transaction should happen correctly", async () => {
+  test.only("Transaction should happen correctly", async () => {
```

Comment:
should remove only

Reply:
actually the entire tests for Transfer page using imported account is skipped at line 206.